contributing.md

```diff
@@ -0,0 +1,70 @@
+# Contributing to {{project-name}}
+
+We welcome everyone to contribute what they can. Whether you are brand new, just want to contribute a little bit, or want to contribute a lot there is probably something you can help out with. Check out our [good first issues](https://build.prestashop-project.org/news/a-definition-of-the-good-first-issue-label/) label for in the issues tab to see a list of issue that good for those new to the project.
+
+## Where to Get Help
+The main way to get help is on our [discord server](https://discord.gg/uh69TdKfBD).  Though, this guide should help you get started. It may be slightly lengthy, but it's designed for those who are new so please don't let length intimidate you.
+
+## Code of Conduct
+Please be kind, inclusive, and considerate when interacting when interacting with others and follow our [code of conduct](./code_of_conduct.md).
+
+## How to Contribute
+If the code adds a feature that is not already present in an issue, you can create a new issue for the feature and add the pull request to it. If the code adds a feature that is not already present in an issue, you can create a new issue for the feature and add the pull request to it.
+
+### Contributing by Adding a Topic for Discussion
+#### Issues
+If you have found a bug and would like to report it or if you have a feature that you feel we should add, then we'd love it if you opened an issue! ❤️ Before you do, please search the other issues to avoid creating a duplicate issue. 
+To submit a new issue just hit the issue button and a choice between two templates should appear. Then, follow along with the template you chose. If you don't know how to fill in all parts of the template go ahead and skip those parts. You can edit the issue later.
+
+#### Discussion
+If you have some new discussion you want to start but it isn't a bug or feature add, then you can start a discussion. Some examples of what kinds of things that are good discussion topics can include, but not limited to the following
```

Comment:
```suggestion
If you have a new discussion you want to start but it isn't a bug or feature add, then you can start a [GitHub discussion](https://docs.github.com/en/discussions). Some examples of what kinds of things that are good discussion topics can include, but are not limited to the following
```

GitHub discussions are fairly new, so it might be good to link out to the docs for them.